src/identity/hcs/vc/hcs-vc.ts

```diff
@@ -63,10 +81,52 @@ export class HcsVc {
      * @returns this
      */
     private async submitTransaction(
-        VcMethodOperation: any,
-        event: any,
-        privateKey: PrivateKey
-    ): Promise<MessageEnvelope<any>> {
-        throw new Error('Method not implemented.');
+        VcMethodOperation: VcMethodOperation,
+        credentialHash: string,
+        issuerPrivateKey: PrivateKey
+    ): Promise<MessageEnvelope<HcsVcMessage>> {
+        const message = new HcsVcMessage(VcMethodOperation, credentialHash);
+        const envelope = new MessageEnvelope(message);
+        const transaction = new HcsVcTransaction(envelope, this._topicId);
+
+        return new Promise((resolve, reject) => {
+            transaction
+                .signMessage((msg) => {
+                    return issuerPrivateKey.sign(msg);
+                })
+                .buildAndSignTransaction((tx) => {
+                    return tx
+                        .setMaxTransactionFee(HcsVc.TRANSACTION_FEE)
+                        .freezeWith(this.client)
+                        .sign(this.accountPrivateKey);
+                })
+                .onError((err) => {
+                    // console.error(err);
+                    reject(err);
+                })
+                .onMessageConfirmed((msg) => {
+                    if (this.onMessageConfirmed) {
+                        this.onMessageConfirmed(msg);
+                    }
+
+                    console.log("Message Published");
+                    console.log(`Explore on DragonGlass: https://testnet.dragonglass.me/hedera/topics/${this.topicId}`);
```

Comment:
It would be good to have this url somewhere in the constants to be reusable and easy to change in one place, what do you think?